librz/core/cmd/cmd_print.c

```diff
@@ -3966,6 +3967,46 @@ RZ_IPI RzCmdStatus rz_cmd_disassembly_function_handler(RzCore *core, int argc, c
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_print_function_rzil_handler(RzCore *core, int argc, const char **argv) {
+	ut64 start, end, size, oldoff;
+	RzAnalysisFunction *fcn;
+	RzList *list;
+
+	oldoff = core->offset;
+	list = rz_analysis_get_functions_in(core->analysis, core->offset);
+	if (rz_list_empty(list)) {
+		RZ_LOG_ERROR("No function found in 0x%08" PFMT64x ".\n", core->offset);
+		goto exit;
+	}
+	if (rz_list_length(list) > 1) {
+		RZ_LOG_ERROR("Multiple overlapping functions found at 0x%08" PFMT64x ". "
```

Comment:
```suggestion
		RZ_LOG_ERROR("Multiple overlapping functions found at 0x%" PFMT64x ". "
```
Better not limit the address to 8 digits. Address spaces with >32bit are not uncommon.